engine/src/Window/Window.cpp

```diff
@@ -90,11 +102,20 @@ namespace Wraith
         if (width == 0 || height == 0)
             return;
 
-        auto* window = (Window*)glfwGetWindowUserPointer(handle);
+        auto window = (Window*)glfwGetWindowUserPointer(handle);
         window->m_Resolution = { (u32)width, (u32)height };
-        for (const auto& [h, callback] : window->s_ResizeCallbacks)
+        for (const auto& [h, callback] : s_ResizeCallbacks)
         {
             callback(width, height);
         }
     }
+    void Window::HandleContentScale(GLFWwindow* handle, float scaleX, float scaleY)
+    {
+        auto window = (Window*)glfwGetWindowUserPointer(handle);
+        window->m_ContentScale = { scaleX, scaleY };
+        for (const auto& [h, callback] : s_ContentScaleCallbacks)
+        {
+            callback(scaleX, scaleY);
+        }
+    }
```

Comment:
As we add more of these, we should consider implementing our own event system, since adding a new array + a couple functions every time we want to add callbacks could become somewhat bloated.

Reply:
#37